### DISCUSSION TITLE
Force slash command refreshing

### DISCUSSION THREAD

**Discussion by vlourme** [tandemdude/hikari-lightbulb#219] [Q&A] [answered]
Hello,

It there a way to force slash command synchronization?

Thanks

**Comment by tandemdude** [comment, marked as answer]
Apologies for missing this! There is a method available on the `BotApp` class: `sync_application_commands` which you can call in order to force a sync of all application commands registered to the bot with discord.
See [Here](https://hikari-lightbulb.readthedocs.io/en/latest/api_references/app.html#lightbulb.app.BotApp.sync_application_commands)
